spec/rubocop/cop/rails/reversible_migration_spec.rb

```diff
@@ -110,6 +110,16 @@ def change
     RUBY
   end
 
+  context 'change_column' do
+    it_behaves_like 'offense', 'change_column', <<~RUBY
+      change_column(:posts, :state, :string)
+    RUBY
+
+    it_behaves_like 'offense', 'change_column', <<~RUBY
+      change_column(:posts, :state, :string, null: false)
+    RUBY
```

Comment:
Can you add an `accepts` test case for `change_column`?

Reply:
Added